src/helpers/n-formatter.pipe.ts

```diff
@@ -1,32 +0,0 @@
-import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
Following my comment about **avoiding  DOM manipulation** (see above), removing the pipe is not good, as ideally it could be used in the template for share-button